### DISCUSSION TITLE
Save/export directly to a server

### DISCUSSION THREAD

**Discussion by yairz83** [GetPublii/Publii#1522] [General]
I love publii !

I think it's a genius concept, and I'm glad to see the project moving forward. I have a question that I would like to share with you. I plan to set up a linux server at home that will run/store many sites including publii sites. What is the most recommended way in terms of exporting the files to the server? I suppose FTP could work, but I thought it might be simpler to set the default directory - which would be located directly on the server (network folder...).
I'm just not that experienced in these things and I'm interested in how people deal with such issues, I hope I'm not the first :)
Thanks!

**Comment by riker09** [comment]
I'm currently in the middle of figuring this out for myself. I just have to deal with one site, though. My plan is to use (S)FTP for the actual upload, as I don't see anything wrong with that. The Publii files (including the database) are going to be shared between approx. 3 persons via some sort of sync mechanism (Resilio, DropBox, ...?).

If you have all your servers in one network I suppose it is easier to just use a shared folder (SMB, NFS) and "upload" directly. My hosting is remote, so that's not a viable option for me.
